src/app/dashboard/users/page.tsx

```diff
@@ -133,14 +154,41 @@ async function getUsers(
     }),
   )
 
-  users.map((user, i) => {
+  listedUsers.map((user, i) => {
     user.requests = usersRequestsCounts[i]?.requests || 0
     user.movies_plays_count = usersPlaysAndDurations[i].movies_plays_count
     user.shows_plays_count = usersPlaysAndDurations[i].shows_plays_count
     user.audio_plays_count = usersPlaysAndDurations[i].audio_plays_count
   })
 
-  return users
+  return listedUsers
+}
+
+async function getStatsWithLoggedInUser(
+  settings: Settings,
+  users: TautulliUserItemRow[],
+  numberOfUsers: number,
+) {
+  let listedUsers = users.slice(0, numberOfUsers)
+  const session = await getServerSession(authOptions)
+  const userId = session?.user?.id
+
+  if (settings.general.isOutsideAccess && !userId) {
```

Comment:
`or` wouldnt work, since this is for checking if the current session is an anonymous one. With `or` we would always return prematurly if the outside access is enabled, even when there is currently a logged in users